pkgs/development/tools/frink/default.nix

```
@@ -0,0 +1,33 @@
+{ fetchurl, jdk, lib, rlwrap, stdenv }:
+stdenv.mkDerivation rec {
+  name = "frink";
```

```suggestion
  pname = "frink";
  version = "unstable-2023-01-28";
```

This is the date provided by Web Archive, from the `/web/20230128164131/` substring.